pages/configuration/https.rst

```diff
@@ -66,11 +66,10 @@ PKCS #12 key stores (PFX files) are commonly used on Microsoft Windows.
 
 In this example, the PKCS #12 (PFX) file is named ``keystore.pfx``::
 
-  $ openssl pkcs12 -in keystore.pfx -nokeys -out graylog-certificate.pem
-  $ openssl pkcs12 -in keystore.pfx -nocerts -out graylog-pkcs5.pem
-  $ openssl pkcs8 -in graylog-pkcs5.pem -topk8 -out graylog-key.pem
+  $ openssl pkcs12 -nokeys -in keystore.pfx -out graylog.crt
+  $ openssl pkcs12 -nocerts -nodes -in keystore.pfx -out graylog.key
 
-The resulting ``graylog-certificate.pem`` and ``graylog-key.pem`` can be used in the Graylog configuration file.
+The resulting ``graylog.crt`` and ``graylog.key`` can be used in the Graylog configuration file. You'll use these files in the Nginx directory. 
```

Comment:
could you please rewrite this that this makes more sense.

why should someone put the files in the nginx directory? What is the benefit?

Please think about what Information someone would need not having your knowledge to understand this sentence.